core/shared/platform/common/posix/posix_thread.c

```diff
@@ -382,6 +386,10 @@ os_thread_get_stack_boundary()
     if ((addr = (uint8 *)pthread_get_stackaddr_np(self))) {
         stack_size = pthread_get_stacksize_np(self);
 
+#ifdef WASM_STACK_GUARD_SIZE
```

Comment:
The macro is always defined, how about removing macro control here?

Reply:
Changed to `#if WASM_STACK_GUARD_SIZE == 0`